test/publish.test.js

```diff
@@ -1,11 +1,26 @@
 import test from 'ava';
 import nock from 'nock';
+import tempy from 'tempy';
 import {stub} from 'sinon';
 import publish from '../lib/publish';
 import authenticate from './helpers/mock-gitlab';
 
 /* eslint camelcase: ["error", {properties: "never"}] */
 
+const owner = 'test_user';
+const repo = 'test_repo';
+const env = {GITLAB_TOKEN: 'gitlab_token'};
+const pluginConfig = {};
+const nextRelease = {gitHead: '123', gitTag: '@scope/v1.0.0', notes: 'Test release note body'};
+const options = {repositoryUrl: `https://gitlab.com/${owner}/${repo}.git`};
+const encodedRepoId = encodeURIComponent(`${owner}/${repo}`);
+const encodedGitTag = encodeURIComponent(nextRelease.gitTag);
+const releaseUrl = `/projects/${encodedRepoId}/repository/tags/${encodedGitTag}/release`;
+const releaseBody = {
```

Comment:
Should fit on one line

Reply:
that was moved, but a change to one line